skbio/stats/distance/_base.py

```diff
@@ -1476,6 +1586,36 @@ def __getitem__(
             else:
                 return self._data.__getitem__(index)  # type: ignore[index]
 
+    def as_redundant(self) -> "SymmetricMatrix":
+        """Return a redundant form copy of the matrix.
+
+        Returns
+        -------
+        SymmetricMatrix
+            A new matrix object with the same data stored in redundant form.
+
+        Notes
+        -----
+        This method always returns a new object, even if the matrix is already
```

Comment:
This sentence may be confusing. First, the new object should be a new matrix object, since every Python variable is an object. Second, does this process copy the underlying data? This is not immediately clear. The source code seems to indicate that everything will be copied. Third, always returns a new object may be a good behavior, because numpy's `astype` also does that. The latter has a `copy` parameter to control the behavior though. Adding `copy` isn't a current priority though.